src/extension/extensionHost.ts

```diff
@@ -82,6 +83,9 @@ function createExtensionHandle(initData: InitData, connection: Connection): type
     const languageFeatures = new ExtLanguageFeatures(proxy('languageFeatures'), documents)
     handleRequests(connection, 'languageFeatures', languageFeatures)
 
+    const searchFeatures = new ExtSearch(proxy('searchFeatures'))
```

Comment:
You call it "search" everywhere else, not "searchFeatures". So for consistency, replace `searchFeatures` with `search` here and on the next line (in both the var name and string). This does not affect the behavior, it just makes it clearer and avoids the question "why was the term 'features' used here?"